includes/admin/post-types/meta-boxes/class-ph-meta-box-appraisal-property-owner.php

```diff
@@ -250,7 +250,7 @@ public static function output( $post ) {
 
         if (keyword.length < 3)
         {
-            $('#appraisal_search_property_owner_results').html('<div style="padding:10px;">Enter ' + (3 - keyword.length ) + ' more characters...</div>');
+            $('#appraisal_search_property_owner_results').html('<div style="padding:10px;"><?php echo esc_html__( 'Enter ', 'propertyhive' ); ?>' + (3 - keyword.length ) + '<?php echo esc_html__( ' more characters...', 'propertyhive' ); ?></div>');
```

Comment:
I don't think the space should part of the esc_html__() call. I think it should just pass 'Enter' instead of 'Enter '. Likewise ' more characters...' should pass just 'more characters' (no space and no '...').

Reply:
Hello @propertyhive 

if you confirm that this is fine, I will make changes to all the other occurrences.

Thanks.